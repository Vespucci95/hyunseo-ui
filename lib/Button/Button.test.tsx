import * as React from 'react';
import { Button } from '../index';
import { render, screen } from '@testing-library/react';

describe('<Button>', () => {
  it('renders component', () => {
    expect(1).toBe(1);
    const { container } = render(<Button>Button</Button>);
    const text = screen.getByText('Button');
    expect(text).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
