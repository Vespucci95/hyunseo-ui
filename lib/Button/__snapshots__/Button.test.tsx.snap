// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button> renders component 1`] = `
<div>
  <button>
    Button
  </button>
</div>
`;
